python/lsst/dax/distribution/DataIngest.py

```diff
@@ -0,0 +1,297 @@
+        print("base_url=", self._base_url)
+
+    def __repr__(self):
+        out = ("host=" + self._host + ":" + str(self._port)
+             + " user=" + self._user + "auth_key:****")
+        return out
+
+    def isIngestAlive(self):
+        """Test if ingest system is alive.
+        """
+        url = self._base_url + 'meta/version'
+        print("ingest url=", url)
+        response = requests.get(url)
+        r_json = response.json()
+        if not r_json['success']:
+            print('error: ' + r_json['error'])
+            return False
+        print('ingest version=', r_json['version'])
+        return True
+
+    def _postToIngest(self, ingest_cmd, data_json):
+        """Send the data_json to the ingest system and return
+        a json result
+        """
+        url = self._base_url + ingest_cmd
+        print('url=', url, " data=", data_json)
+        response = requests.post(url, json=data_json)
+        r_json = response.json()
+        success = True
+        if not r_json['success']:
+            print('ERROR post url=', url, " data=", data_json,
+                  'error: ', r_json['error'])
+            success = False
+        return success, r_json
+
+    def _putToIngest(self, ingest_cmd, data_json):
+        """Send the data_json to the ingest system and return
+        Return
+        ------
+        success : bool
+            True indicates operation was successful
+        status_code : int
+            Status code value from put request.
+        r_json : json or None
+            If the status code was not 200, this will be None.
+            Otherwise it is a json object with information about the request.
+        """
+        url = self._base_url + ingest_cmd
+        print('url=', url, " data=", data_json)
+        response = requests.put(url, json=data_json)
+        status_code = response.status_code
+        success = True
+        # 200 means the put request was at least well formed.
+        if status_code != 200:
+            print('ERROR put url=', url, "data=", data_json, 'status=', status_code)
+            success = False
+            return success, status_code, None
+        r_json = response.json()
+        if not r_json['success']:
+            print('ERROR put url=', url, 'status=', status_code, 'data=', data_json,
+                  'r_json=', r_json)
+            success = False
+        return success, status_code, r_json
+
+    def sendDatabase(self, db_file_path):
+        """ Send the database description to the ingest system.
+        """
+        with open(db_file_path, 'r') as db_f:
+            data = db_f.read()
+            data_json = json.loads(data)
+        success, r_json = self._postToIngest('ingest/database', data_json)
+        if not success:
+            print('ERROR while sending databaae ', db_file_path, "r_json=", r_json)
+            return False
+        return True
+
+    def sendTableSchema(self, schema_file_path):
```

Comment:
See my comment for method `sendDatabase`. This method should be named as `registerTable` or `createTable`.